emulators/ppsspp.md

```diff
@@ -0,0 +1,46 @@
+# PPSSPP Guide
+
+❗ Follow the [Getting Started section](../README.md#getting-started) of `README.md` first.
+
+This section of the guide explains how to setup [PPSSPP](https://www.ppsspp.org), a PSP emulator.
+
+![image](https://user-images.githubusercontent.com/59558433/163827712-fe70ad3e-b625-4439-96ea-40038c128a00.png)
+
+## Installing PPSSPP
+
+Open Discover, SteamOS' app store, then search for **PPSSPP**. Click on the one labeled "A PlayStation Portable emulator"
+
+In the top right, press **Install**.
+
+Alternatively, open up a terminal and run
+
+```bash
+flatpak install --user -y org.ppsspp.PPSSPP
+```
+
+## Configuring PPSSPP
+
+Open up the emulator and press **Settings**.
+
+In **Graphics**:
+ * select **Vulkan** as a Backend and say *Yes* to restart now, then launch PPSSPP again.
+ * Go back in Settings, Graphics, then set **AMD RADV VANGOGH** as a Device.
+ * Turn ON **Fullscreen**, **VSync** and set the **Rendering resolution** to **Auto (1:1)**
+ * Set **Upscale level** to **2x** and **Upscale type** to **Hybrid + Bicubic**, then check **Deposterize**
+
+Go back and select *Exit*, Launch it one last time and in the **Games** tabs on main menu, navigate to wherever your PSP ROMs are - if you used the recommended path, this should be `~/roms/psp`.
+
+## Steam ROM Manager
+
+Open Steam ROM Manager, press Parsers, then enter the following settings:
+
+-   Select `Sony PlayStation Portable - PPSSPP (Flatpak)` under Community Presets.
+-   You can add additional categories to Steam category using the format `${category name}`, this is case-sensitive.
+-   Set executable to `/usr/bin/flatpak`.
+-   Set command line arguments to `run org.ppsspp.PPSSPP "${filePath}"`
```

Comment:
Unless I'm missing something, I don't think this is needed; the PPSSPP Flatpak preset already has this filled in.

Reply:
100% but so does the dolphin flatpak and it was specified, i though it was on purpose do i did the same :)
I can remove it

Reply:
The Dolphin guide doesn't use the Flatpak yet.

Reply:
And that show how good i am at reading :flushed: 
thank's !